disco_aws_automation/disco_elastigroup.py

```diff
@@ -250,8 +281,8 @@ def update_group(self, hostclass, desired_size=None, min_size=None, max_size=Non
             block_device_mappings=block_device_mappings,
             group_name=group_name or self._get_new_groupname(hostclass)
         )
-        group = self.get_existing_group(hostclass)
-        if group:
+        group = self.get_existing_group(hostclass, group_name)
+        if group and not create_if_exists:
```

Comment:
A bit confused about `create_if_exists`. Why would we create a group if it already exists